.travis.yml

```diff
@@ -56,5 +57,4 @@ deploy:
   distributions: "sdist bdist_wheel"
   on:
     tags: true
-    python: "2.7"
-    condition: $REQUIREMENTS = release
```

Comment:
I think this is needed to distinguish the requirements of the released package.

Reply:
@Glignos again I removed it due to the reference in the [RFC](https://github.com/inveniosoftware/rfcs/blob/master/framework/0016-python2.md)

Reply:
@topless I think I was mistaken since the context of the `$REQUIREMENTS` is limited to the testing. So this should be fine indeed. :)

Reply:
Disccued with RFC's creator (Lars) the references to condition and python are not needed. Following the RFC - as is- is correct. Sorry for the misunderstanding.

```diff
-python: "2.7
-condition: $REQUIREMENTS = release
```